service/src/main/resources/application.yaml

```diff
@@ -8,7 +8,9 @@ management:
   endpoint:
     health:
       show-details: "always"
-
+  health:
+    probes:
+      enabled: false
```

Comment:
new 2.3.0 feature that collides with 
uk.gov.hmcts.reform.health-spring-boot-starter: '0.0.5'

Reply:
There has been [some discussion](https://hmcts-reform.slack.com/archives/C8SR5CAMU/p1589978499323600) around this already, don't know if it is helpful

Reply:
good to know, was not aware about this discussion.